### DISCUSSION TITLE
Crafting Custom Connectors with Airbyte's Low-Code Magic: A Step-by-Step Guide

### DISCUSSION THREAD

**Discussion by parthiv11** [airbytehq/airbyte#29644] [Show and tell]
Hey there, lovely people!

I hope this message finds you in good spirits👋.I wanted to reach out and share my recent blog post, where I take a deep dive into the fascinating world of building new connectors in Airbyte using the Connector Builder UI🛠️✨. In the blog, I provide a detailed, step-by-step guide on how to create connectors without needing any coding experience. Yes, you read that right – no coding required! With just a few interactions with input fields and buttons, you can develop connectors within minutes.⏰💡

Feel free to check it out here: https://parthiv11.hashnode.dev/airbyte-nocode-magic

I'm excited to hear your thoughts, questions, and experiences related to this topic. Whether you've encountered challenges while building connectors💬🔌 using the Connector Builder UI, or if you have questions about the process outlined in the blog – I'm all ears.

Let's use this opportunity to have an insightful discussion and address any doubts you might have about the Connector Builder UI. After all, we're in this together to make connector development more accessible and enjoyable for everyone.

Looking forward to engaging with each and every one of you. 🚀

**Comment by saulBurgosProtiv** [comment]
thanks you

**Comment by philipandersson** [comment]
If you need help creating low-code connectors, try our prototype for generating them: https://github.com/skyffel/airbyte-connector-generator-poc
